Darkplaces-Emscripten/clean.sh

```diff
@@ -0,0 +1,19 @@
+rm build/darkplaces-emscripten.html
```

Comment:
`makefile.inc` mentions `darkplaces-wasm.html`. Is that a different thing?

Reply:
Yeah I’ll remove that. It’s not a different thing I just forgot about that line.